tests/test_translator.py

```diff
@@ -68,12 +74,53 @@ def test_track_to_mpd_format_for_nonempty_track(self):
         )
         assert ("file", "à uri") in result
         assert ("Time", 137) in result
+        assert ("Artist", "an artist;yet another artist") in result or (
+            "Artist",
+            "yet another artist;an artist",
+        ) in result
```

Comment:
This solution isn't particularly elegant, but I can't think of anything better off the top of my head :shrug: